src/angular-version-comparer.ts

```diff
@@ -0,0 +1,28 @@
+import {AngularVersion} from './types/angular-version';
+
+/**
+ *
+ * Compares two angular versions, A and B. Based on their minor and major versions.
+ *
+ *
+ * @export
+ * @param {AngularVersion} versionA
+ * @param {AngularVersion} versionB
+ * @return {number}  > 0 if A is newer, 0 if they are the same version, < 0 if A is older
+ */
+export function angularVersionComparer(
+  versionA: AngularVersion,
+  versionB: AngularVersion
+): number {
+  // Gets the major and minor versions
+  const [majorA, minorA] = versionA.split('.');
+  const [majorB, minorB] = versionB.split('.');
+
+  // Obtains the major version difference between versions A and B
+  const majorComparison = +majorA - +majorB;
+  // Obtains the minor version difference between versions A and B
+  const minorComparison = +minorA - +minorB;
```

Comment:
suggestion: Use `Number.parseInt` instead of the old school `+` prefix operator. It's especially confusing when combined with arithmetic like the infix `-` operator here.

Reply:
suggestion: Rename the variables to `*Difference` to make the calculations clearer.